pkg/controller/account/byoc.go

```diff
@@ -394,6 +410,54 @@ func DeleteRole(reqLogger logr.Logger, byocRole string, byocAWSClient awsclient.
 	return err
 }
 
+func (r *ReconcileAccount) getSTSClient(log logr.Logger, accountClaim *awsv1alpha1.AccountClaim, operatorAWSClient awsclient.Client) (awsclient.Client, *sts.AssumeRoleOutput, error) {
+	// Get SRE Access ARN from configmap
+	cm := &corev1.ConfigMap{}
+	cmErr := r.Client.Get(context.TODO(), types.NamespacedName{Namespace: awsv1alpha1.AccountCrNamespace, Name: awsv1alpha1.DefaultConfigMap}, cm)
+	if cmErr != nil {
+		log.Error(cmErr, "There was an error getting the ConfigMap to get the STS Jump Role")
+		return nil, nil, cmErr
+	}
+
+	stsAccessARN := cm.Data["STS-Jump-Role"]
+	if stsAccessARN == "" {
+		log.Error(awsv1alpha1.ErrInvalidConfigMap, "configmap key missing", "keyName", "STS-Jump-Role")
+		return nil, nil, cmErr
+	}
+
+	jumpRoleCreds, err := getStsCredentials(log, operatorAWSClient, stsAccessARN, "awsAccountOperator")
```

Comment:
@iamkirkbater this is not what the AAO does today but you can save some lines here an use a provided function from  the AWS SDK [here](https://docs.aws.amazon.com/sdk-for-go/api/aws/credentials/stscreds/#NewCredentials)

Hive are actually using it [here](https://github.com/openshift/hive/pull/1306/files#diff-781a1cdc5ca06b860f587e10c6144a65f58ba703efe886db21d61f5b1fb22978R549) if you want an example. This might be a second PR to refactor.